src/PKSim.Core/Model/Population.cs

```diff
@@ -228,10 +228,7 @@ public virtual IEnumerable<IParameter> AllConstantParameters(IEntityPathResolver
 
       private bool parameterShouldBeReturnedAsConstant(IParameter parameter)
       {
-         if (parameter.IsNamed(CoreConstants.Parameters.REFERENCE_CONCENTRATION))
```

Comment:
Here we were bypassing the implementation because of the wrong implemenation

Reply:
I checked all usage of previous `IsIndividualMolecule`. It will know perform ok when used to genereate Individual pdf (it was not the case before)